runhouse/main.py

```diff
@@ -978,16 +979,14 @@ def server_status(
     """Check the HTTP server status on the cluster."""
     logger.debug("Checking the server status.")
     current_cluster = get_cluster_or_local(cluster_name=cluster_name)
```

Comment:
realizing this now - but if the cluster isn't found `get_cluster_or_local` will throw an exception, maybe we should put that in a try/catch so if there's a ValueError we handle it more gracefully?